### PR TITLE
predicate changes

### PR DESCRIPTION
**Pull request by Jeelan9686**
please review and approve

### CODE REVIEW

File: src/Test.java [hunk 1]
```diff
@@ -2,10 +2,11 @@ public class Test {
 	public static void main(String[] args) {
```
Comment (khadri2021) [inline src/Test.java:2]: remove this test class , its not required for master branch

**Review by khadri2021 — CHANGES_REQUESTED** [review]
do these changes.

File: src/Test.java (hunk 1 above)
Comment (Jeelan9686) [reply src/Test.java]: i removed unused test class

File: src/TestMain.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+import java.util.function.Predicate;
+
+public class TestMain {
+	static Scanner sc = new Scanner(System.in);
```
Comment (khadri2021) [inline src/TestMain.java:5]: define private

File: src/TestMain.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+	public static void main(String[] args) {
+
+		System.out.println("WELCOME TO TEST MAIN....");
+		System.out.println("");
```
Comment (khadri2021) [inline src/TestMain.java:10]: remove sysout which is not required.

File: src/TestMain.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+		};
+		boolean name1 = pr.test(name);
+		System.out.println("your NAME " + name + " IS ==>" + name1);
+
+		Predicate<Integer> pr1 = (i) -> {
+			return i > 0;
+		};
+		boolean result1 = pr1.test(id);
+		System.out.println("your ID " + id + " IS ==>" + result1);
+		System.out.println();
```
Comment (khadri2021) [inline src/TestMain.java:31]: remove this sysout which is not required.

File: src/Testpojo.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
+
+public class Testpojo {
+	String name;
```
Comment (khadri2021) [inline src/Testpojo.java:3]: define properties with private

File: src/predicateandfunction/RegId.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
+package predicateandfunction;
+
+public class RegId {
+	String regId;
```
Comment (khadri2021) [inline src/predicateandfunction/RegId.java:4]: define properties with private

File: src/predicateandfunction/Register.java [hunk 7]
```diff
@@ -0,0 +1,21 @@
+package predicateandfunction;
+
+public class Register {
+
+	String name;
```
Comment (khadri2021) [inline src/predicateandfunction/Register.java:5]: define properties with private

**Review by khadri2021 — COMMENTED** [review]
first review

File: src/TestMain.java (hunk 2 above)
Comment (Jeelan9686) [reply src/TestMain.java:5]: yes i do sir

File: src/TestMain.java (hunk 3 above)
Comment (Jeelan9686) [reply src/TestMain.java:10]: ok sir

File: src/TestMain.java (hunk 4 above)
Comment (Jeelan9686) [reply src/TestMain.java:31]: ok sir

File: src/Testpojo.java (hunk 5 above)
Comment (Jeelan9686) [reply src/Testpojo.java:3]: yes i do sir

File: src/predicateandfunction/RegId.java (hunk 6 above)
Comment (Jeelan9686) [reply src/predicateandfunction/RegId.java:4]: ok sir

**Comment by khadri2021** [comment]
done.

**Pull request closed without merging**